data/src/main/java/ca/bc/gov/data/local/entity/PatientEntity.kt

```diff
@@ -24,7 +25,9 @@ data class PatientEntity(
     @ColumnInfo(name = "time_stamp")
     val timeStamp: Instant = Instant.now(),
     @ColumnInfo(name = "patient_order", defaultValue = Long.MAX_VALUE.toString())
-    val patientOrder: Long
+    val patientOrder: Long,
+    @ColumnInfo(name = "authentication_status", defaultValue = "")
```

Comment:
Please mention the defaultvalue = "". The value of AuthenticationStatus.NON_AUTHENTICATED needs to placed here instead of empty string so that existing records are also updated.